src/createApp.ts

```diff
@@ -373,7 +373,7 @@ export interface ReadOnlyRegistry {
 	 * @param id Identifier for the widget
 	 * @return `true` if a widget has been registered, `false` otherwise.
 	 */
-	hasWidget(id: Identifier): boolean;
+	hasWidget(id: Identifier): Promise<boolean>;
```

Comment:
Why do we need to do this in a Promise? (Also comment now out of sync with code)